WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -332,6 +332,13 @@ - (void)checkEligibility:(void (^)(BOOL))eligible {
             [WTRLogger log:@"User eligible. Code: %@. Description: %@.", responseJSON[@"details"][@"code"], responseJSON[@"details"][@"why"]];
 
             [self->_settings parseDataFromSurveyServer:responseJSON];
+            self->_accountToken = responseJSON[@"settings"][@"account_token"];
+            self->_clientID = responseJSON[@"settings"][@"client_id"];
```

Comment:
@diegoserranoa Do you expect `client_id` to be present all time in the eligibility response?

Reply:
@prabhatjha nope, which is why I'm checking if `_accountToken` or `_clientID` is `null` at line [338](https://github.com/Wootric/WootricSDK-iOS/pull/75/files#diff-1fcc1204f79424831eb0db041dc5bebeR338l) and returning an error if any of those values are not present.

Reply:
I missed the part that eligibility only returns `client_id` for mobile SDK calls. All good now.